src/main/java/dk/creditoro/client/model/crud/Production.java

```diff
@@ -0,0 +1,31 @@
+package dk.creditoro.client.model.crud;
+
+public class Production {
+    private String identifier;
+    private String title;
+    private String[] producers;
```

Comment:
A production can only have 1 producer.

Reply:
oh, so the API returns a producer object?